fs/copy_linux.go

```diff
@@ -43,28 +43,41 @@ func copyFileInfo(fi os.FileInfo, name string) error {
 	return nil
 }
 
+const maxCopySize = 2147483647 // max 32bit integer
```

Comment:
Could we just use `math.MaxInt32` here?

Reply:
Certainly. Didn't want to add the import just to get one number.

Reply:
Updated